saleor/app/utils.py

```diff
@@ -0,0 +1,19 @@
+from typing import Union
+
+from django.contrib.auth.hashers import check_password
+
+from saleor.app.models import App, AppToken
+
+
+def get_app(raw_auth_token) -> Union[None, App]:
```

Comment:
Is this function used anymore? I'm struggling to find its usages or I'm blind ;) There is a bunch of warning that it is untested.

Reply:
Right now it might not be used anymore, I'll double-check it, thanks.

Reply:
Removed.